rs/bitcoin/kyt/src/main.rs

```diff
@@ -94,6 +95,23 @@ fn post_upgrade(arg: KytArg) {
     }
 }
 
+#[ic_cdk::query(hidden = true)]
+fn http_request(req: http::HttpRequest) -> http::HttpResponse {
+    if req.path() == "/metrics" {
+        // TODO: metrics
```

Comment:
nit: Maybe having a ticket for this would be useful so that we can note down some idea, e.g.,having a metric for the number of entries in the cache.

Reply:
Added a ticket number.

Reply:
I added a few bullet points as metrics that could be relevant. This is only after 5s brainstorming, so feel free to change/add/challenge any of those.